modules/deploy/pages/deployment-option/cloud/configure-privatelink-private-service-connect-in-cloud-ui.adoc

```diff
@@ -0,0 +1,88 @@
+= Configure PrivateLink and Private Service Connect in the Cloud UI
+:description: Set up AWS PrivateLink and GCP Private Service Connect in the Redpanda Cloud UI.
+:page-cloud: true
+
+include::shared:partial$feature-flag.adoc[]
+
+NOTE: This guide is for configuring AWS PrivateLink and GCP Private Service Connect using the Redpanda Cloud UI. See xref:deploy:deployment-option/cloud/aws-privatelink.adoc[Configure AWS PrivateLink for Redpanda Cloud] or xref:deploy:deployment-option/cloud/gcp-private-service-connect.adoc[Configure GCP Private Service Connect for BYOC] if you want to set up these services using the API.
```

Comment:
I think it would be better to keep them separate. The overlap in process, requirements & terminology is almost none between GCP & AWS, and Azure Private Link (notice the space, differing from AWS's PrivateLink) is underway.
Having 3 guides combined into one seems like a lot.

Reply:
@0x5d separated into two docs, thank you!